### DISCUSSION TITLE
fast-dreambooth local installation?

### DISCUSSION THREAD

**Discussion by ZeroCool22** [TheLastBen/fast-stable-diffusion#252] [Q&A]
I only see the Colab one...

**Comment by ZeroCool22** [comment]
Any news about this?

![nmnmnnn](https://user-images.githubusercontent.com/13344308/198887747-d2c96a14-235d-4f74-ada1-51ae4e6aa907.gif)

**Comment by TheLastBen** [comment]
I need a beta tester

https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/Local_fast_DreamBooth-Win.ipynb

**Reply by NeptuneTQ** [reply]
So I tried to run this, and this is the error message I get when just trying to install dependencies.  I have the Automatic SD WebUI already installed on my pc.  Should I be running the Local_Dreambooth from a certain folder on my PC?  Also, where should I be putting the checkpoint file I want to train, or is that automatically downloaded at some point?

Cloning into 'diffusers'...
WARNING: Skipping transformers as it is not installed.

Requirement already satisfied: py7zr in c:\users\son\appdata\local\programs\python\python310\lib\site-packages (0.20.0)
Requirement already satisfied: pybcj>=0.6.0 in c:\users\son\appdata\local\programs\python\python310\lib\site-packages (from py7zr) (1.0.1)
Requirement already satisfied: inflate64>=0.3.0 in c:\users\son\appdata\local\programs\python\python310\lib\site-packages (from py7zr) (0.3.1)
Requirement already satisfied: multivolumefile>=0.2.3 in c:\users\son\appdata\local\programs\python\python310\lib\site-packages (from py7zr) (0.2.3)
Requirement already satisfied: pycryptodomex>=3.6.6 in c:\users\son\appdata\local\programs\python\python310\lib\site-packages (from py7zr) (3.15.0)
Requirement already satisfied: psutil in c:\users\son\appdata\local\programs\python\python310\lib\site-packages (from py7zr) (5.9.2)
Requirement already satisfied: texttable in c:\users\son\appdata\local\programs\python\python310\lib\site-packages (from py7zr) (1.6.4)
Requirement already satisfied: brotli>=1.0.9 in c:\users\son\appdata\local\programs\python\python310\lib\site-packages (from py7zr) (1.0.9)
Requirement already satisfied: pyzstd>=0.14.4 in c:\users\son\appdata\local\programs\python\python310\lib\site-packages (from py7zr) (0.15.3)
Requirement already satisfied: pyppmd<0.19.0,>=0.18.1 in c:\users\son\appdata\local\programs\python\python310\lib\site-packages (from py7zr) (0.18.3)
100% [........................................................................] 41126563 / 411265630 File(s) copied

File not found - $MAIN_DIR/usr/local/lib/python3.7/dist-packages

Done!

**Reply by TheLastBen** [reply]
put the file in any folder you want, and you need to download the diffusers model manually

**Comment by TheLastBen** [comment]
try it and we'll see
https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/Local_fast_DreamBooth-Win.ipynb

**Reply by ZeroCool22** [reply]
> try it and we'll see https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/Local_fast_DreamBooth-Win.ipynb

I don't think i will be able to run it with 1080TI...

**Comment by ZeroCool22** [comment]
I put my Token here before running the cell?

"http://localhost:8888/?token=xxxxxx"

**Reply by TheLastBen** [reply]
follow the instructions written in the dependencies cell

**Reply by TheLastBen** [reply]
[the token ](https://research.google.com/colaboratory/local-runtimes.html)
